### PR TITLE
[FJD-200] Create an Accordion block

### PR DESCRIPTION
**Pull request by MamoruFivejars**
Jira issue: https://fivejars.atlassian.net/browse/FJD-200
Steps to review in DEMO PR: https://github.com/fivejars/booster_drupal/pull/99

Test on demo environment from DEMO PR

### CODE REVIEW

File: recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion.default.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    third_party_settings: {  }
+  info:
+    type: string_textfield
+    weight: -5
+    region: content
+    settings:
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+  langcode:
```
Comment (OleksandrRiumshyn) [inline recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion.default.yml:40]: Could be removed

File: recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion_item.default.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  info:
+    type: string_textfield
+    weight: -5
+    region: content
+    settings:
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    label: Test
+  langcode:
```
Comment (OleksandrRiumshyn) [inline recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion_item.default.yml:33]: Could be removed

**Review by OleksandrRiumshyn — CHANGES_REQUESTED** [review]

File: recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion.default.yml (hunk 1 above)
Comment (MamoruFivejars) [reply recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion.default.yml]: Done

File: recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion_item.default.yml (hunk 2 above)
Comment (MamoruFivejars) [reply recipes/accordion_block_type/config/core.entity_form_display.block_content.accordion_item.default.yml]: Done

File: recipes/accordion_block_type/config/field.field.block_content.accordion_item.body.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+label: Body/Answer
+description: ''
+required: true
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings:
+  display_summary: false
+  required_summary: false
+  allowed_formats: {  }
```
Comment (OleksandrRiumshyn) [inline recipes/accordion_block_type/config/field.field.block_content.accordion_item.body.yml:22]: Only basic_html and plain text formats should be allowed

**Review by OleksandrRiumshyn — CHANGES_REQUESTED** [review]

File: recipes/accordion_block_type/config/field.field.block_content.accordion_item.body.yml (hunk 3 above)
Comment (MamoruFivejars) [reply recipes/accordion_block_type/config/field.field.block_content.accordion_item.body.yml]: Done

**Review by OleksandrRiumshyn — APPROVED** [review]

**Pull request merged**